inductiva/simulators/fds.py

```diff
@@ -17,6 +17,7 @@ def run(
         self,
         input_dir: types.Path,
         sim_config_filename: str,
+        post_processing_config: str = None,
```

Comment:
Should it be named as `post_processing_config_filename` to be consistent and more explicit?